### DISCUSSION TITLE
Why Extensions shown blank to me after syncing !

### DISCUSSION THREAD

**Discussion by iJustAnas** [Alex313031/Thorium-Win-AVX2#29] [General]
Hi , i really like Thorium but i cant work without seen my extensions and disabled it , i can only deleted it from the shortcut 
any tips how to make it work i'm on windows ! and also previous versions didnt worked on windows 10 but i tried the last one on windows 11 and it worked but as i said no extension  , thx devs

**Comment by Alex313031** [comment]
@StunningFannee This issue has been fixed with https://github.com/Alex313031/Thorium/commit/8d99f75ec4095425ae0eaa414354865fdab80066 but the latest avx2 release doesnt have it yet. We will be building an M107 release soon with this fix.

The issue is I originally added a patch to Thorium to show ALL extensions, even internal ones, but this also enabled showing Themes, which should never be shown on the extensions page, and throws an error in the JavaScript which blocks the page from rendering. That new patch tells it to show all extensions except ones with the Manifest::TYPE_THEME header inside, which are only used by themes. 

The bug is only triggered when a theme is installed, and I am assuming that when you enabled sync, it installed a theme along with your other extensions as part of the data in your google accounts sync data.
